### PR TITLE
#489 Adding spanish translations

### PR DESCRIPTION
**Pull request by eatskolnikov**
## What's new?
- #489 Included more translations to Spanish
- Updated readme.md

A few notes:

1. The string for "Search open collective" needs to be mapped, the string is missing from the localization files
screen shot 2018-06-05 at 9 51 39 pm

2. You will notice I removed some articles and replaced others. In Spanish, you don't always need the gendered article if it is followed by a noun and it reads better. Also tried to make sure the language was as gender neutral as possible.

3. You will also notice I did not translate words like **backer** and **sponsor**. I felt that for the current context those words would have sounded weird in Spanish so I took the liberty of not translating them.

### CODE REVIEW

File: src/lang/es.json [hunk 1]
```diff
@@ -308,7 +308,7 @@
   "menu.admin": "administrador/a",
   "menu.backer": "backer",
   "menu.attendee": "participante",
-  "menu.fundraiser": "fundraiser",
+  "menu.fundraiser": "recaudación de fondos",
```
Comment (piamancini) [inline src/lang/es.json]: este es una persona, no un evento. Seria el recaudador de fondos, pero como es muy largo, decidimos dejarlo como fundraiser

Comment (eatskolnikov) [reply src/lang/es.json]: Entiendo, voy a ponerlo como estaba

Comment (piamancini) [reply src/lang/es.json]: genial. Quedan traducciones aún o con estas ya estamos?

Comment (eatskolnikov) [reply src/lang/es.json]: Creo que con este cambio ya está pero haré double check antes de mandar el commit.

**Comment by eatskolnikov** [comment]
@piamancini Actualicé el PR

File: src/lang/es.json [hunk 2]
```diff
@@ -1,50 +1,50 @@
...
+  "collective.created": "Su colectivo fue creado con éxito.",
+  "collective.created.description": "Mientras espera la aprobación del host({host}), puede editar su colectivo, subir sus gastos y crear eventos.",
   "collective.donate": "donar",
-  "usercollective.since": "Establecido in {year}",
+  "usercollective.since": "Establecido en {year}",
   "collective.members.admin.title": "{n} {n, plural, one {colaborador} other {colaboradores}}",
   "collective.members.member.title": "{n} {n, plural, one {miembro} other {miembros}}",
   "collective.members.backer.title": "{n} {n, plural, one {backer} other {backers}}",
-  "collective.members.fundraiser.title": "{n} {n, plural, one {fundraiser} other {fundraisers}}",
+  "collective.members.fundraiser.title": "{n} {n, plural, one {recaudación de fondos} other {recaudaciones de fondos}}",
```
Comment (agonzalez95) [inline src/lang/es.json]: Se escapo este @eatskolnikov

File: src/lang/es.json [hunk 3]
```diff
@@ -53,56 +53,56 @@
   "collective.card.stats.backers": "{n, plural, one {backer} other {backers}}",
   "collective.card.stats.yearlyBudget": "presupuesto anual",
   "collective.card.memberOf.count": "{n, plural, one {collective} other {collectives}} backed",
-  "collective.card.stats.totalAmountSent": "contributed",
+  "collective.card.stats.totalAmountSent": "conntribuido",
```
Comment (agonzalez95) [inline src/lang/es.json]: typo

File: src/lang/es.json [hunk 4]
```diff
@@ -53,56 +53,56 @@
...
-  "collective.cover.hostedBy": "Hosted by",
-  "collective.cover.pendingApprovalFrom": "Pending approval from",
+  "host.apply.btn": "Aplica para crear un colectivo",
+  "collective.cover.hostedBy": "Su host es",
+  "collective.cover.pendingApprovalFrom": "Aprobación pendiente por parte de ",
   "collective.stats.totalAmountSent.label": "Monto total donado",
-  "collective.stats.totalAmountRaised.label": "Total amount raised",
-  "expenses.badge.tooltip.pending": "{pending} {pending, plural, one {expense} other {expenses}} pending approval",
+  "collective.stats.totalAmountRaised.label": "Monto total recaudado",
+  "expenses.badge.tooltip.pending": "{pending} {pending, plural, one {gasto pendiete} other {gastos pendientes}} de aprobación",
```
Comment (agonzalez95) [inline src/lang/es.json]: `pendiete` typo

**Comment by eatskolnikov** [comment]
@agonzalez95 Oops! sorry for the typos, looks like my keyboard has issues with the letter n :p

I've updated the PR

**Comment by znarf** [comment]
@piamancini can you check the latest version? Let's merge that this week.

**Comment by eatskolnikov** [comment]
@znarf I'll get to that later today

**Comment by eatskolnikov** [comment]
@znarf sorry it took me so long, dayjob got me hard jajaja

**Comment by eatskolnikov** [comment]
@znarf this PR is conflict free and ready to review/merge

**Comment by eatskolnikov** [comment]
@agonzalez95 @piamancini PR ready

**Comment by Betree** [comment]
Hi @eatskolnikov ! :wave:

This is definitely late, but thank you so much for this PR!
I've rebased it and will merge it as soon as our tests pass.

We're currently revamping our translations system to make it more approachable, would be awesome to have you around to contribute on translations again if you want to :slightly_smiling_face:

**Pull request merged**